nanome/util/string_builder.py

```diff
@@ -5,8 +5,11 @@ def __init__(self):
     def append(self, s):
         self.los.append(str(s))
 
-    def to_string(self):
-        joined = "".join(self.los)
+    def append_string(self, s):
```

Comment:
added this function to reduce unnecessary casts when we know the value is a string.